src/lib/components/Header/Header.scss

```diff
@@ -0,0 +1,67 @@
+.ldaf-header {
+  background: linear-gradient(to right, transparent, #00284d 30%),
+    url("../../assets/header-background.png");
+}
+
+.ldaf-header .ldaf-nav__secondary {
+  .usa-input {
+    border-radius: 5px 0 0 5px;
+  }
+
+  a {
+    color: $primary-light;
+  }
```

Comment:
I think this will override the `:visited` state; I think this makes sense for this case, but just wanted to make sure it was intentional.

Reply:
Yeah the secondary nav links are colored differently, I think to account for the [background]([https://](https://www.figma.com/file/oGKbyCnCRRdNzLYbiags93/LDAF-Component-Library-USWDS-3.0.2?node-id=2196-11223&t=1yfrsbwflqSDvces-0)). Idk if we want to have a `:visited` state for those, but something to make @getpunched aware of if in case we do

Reply:
Sounds good to me!